fdbclient/Tuple.cpp

```diff
@@ -462,7 +517,32 @@ TEST_CASE("fdbclient/Tuple/makeTuple") {
 	ASSERT(t1.getType(5) == Tuple::UTF8);
 	ASSERT(t1.getType(6) == Tuple::NULL_TYPE);
 	ASSERT(t1.getType(7) == Tuple::VERSIONSTAMP);
-	ASSERT(t1.size() == 8);
+	ASSERT(t1.getType(8) == Tuple::USER_TYPE);
+	ASSERT(t1.size() == 9);
+
+	Standalone<StringRef> packed = t1.pack();
```

Comment:
This block doesn't really seem to be a test of `makeTuple`, should we move it to a separate test case?

Reply:
Done.

Reply:
I think this part of the test was meant to be deleted after the new test case was created

Reply:
Yeah fixed.